### ISSUE TITLE
_ERROR_STORAGE_MISSING_INFO message too specific

### ISSUE THREAD

**Issue by crwilcox** [Azure/azure-storage-python#74]
if you make a queue (and likely other things) without a key it will tell you that you failed to provide the account name. This error comes up for more cases than that as evidenced by this.  Consider broadening the error to represent the conditions it is thrown for.

CODE:
queue.QueueService("account", "")

RESULT:
```
    qs = queue.QueueService(account_name, account_key)
  File "\site-packages\azure\storage\queue\queueservice.py"
, line 134, in __init__
    raise ValueError(_ERROR_STORAGE_MISSING_INFO)
ValueError: You need to provide an account name
Press any key to continue . . .
```

**Comment by jehine-MSFT** [comment]
Thanks for the great find Chris! Added this bug fix to the backlog. Should hopefully go out with the next major release.

Thanks again!

**Comment by jehine-MSFT** [comment]
Hello @crwilcox ,

We have released the 0.30 version (#132) which addressed this feature request. If you have issues with the new version in regards to this issue, please reopen it and continue the discussion.

Thanks!
